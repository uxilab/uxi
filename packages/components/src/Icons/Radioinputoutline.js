import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Radioinputoutline = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 15.75 15.75"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M7.88,0a7.88,7.88,0,1,0,7.88,7.88A7.88,7.88,0,0,0,7.88,0Zm0,14.22a6.35,6.35,0,1,1,6.35-6.35A6.35,6.35,0,0,1,7.88,14.22Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Radioinputoutline.propTypes = {
  color: PropTypes.string,
};

Radioinputoutline.defaultProps = {
  color: '#6d6d71',
};

export default Radioinputoutline;
