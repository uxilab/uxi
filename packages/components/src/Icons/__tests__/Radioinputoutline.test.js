import React from 'react';
import Radioinputoutline from '../Radioinputoutline';

describe('Icon : <Radioinputoutline />', () => {
  it('match snapshot', () => {
    expect(shallow(<Radioinputoutline />)).toMatchSnapshot();
  });
});

