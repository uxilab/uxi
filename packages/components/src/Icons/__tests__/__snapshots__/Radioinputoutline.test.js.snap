// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Radioinputoutline /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 15.75 15.75"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M7.88,0a7.88,7.88,0,1,0,7.88,7.88A7.88,7.88,0,0,0,7.88,0Zm0,14.22a6.35,6.35,0,1,1,6.35-6.35A6.35,6.35,0,0,1,7.88,14.22Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
